test/Microsoft.Azure.WebJobs.Host.UnitTests/Loggers/DefaultTelemetryClientFactoryTests.cs

```diff
@@ -26,5 +26,15 @@ public void InitializeConfiguguration_Configures()
             // Verify Channel
             Assert.IsType<ServerTelemetryChannel>(config.TelemetryChannel);
         }
+
+        [Fact]
+        public void InitializeConfiguration_ConfiguresWithIngestionEndpoint()
+        {
+            var testIngestionEndpoint = "https://testIngestion.Endpoint/";
+            var factory = new DefaultTelemetryClientFactory(string.Empty, null, testIngestionEndpoint, null, null);
+            var config = factory.InitializeConfiguration();
+
+            Assert.Equal(testIngestionEndpoint.ToLower(), config.TelemetryChannel.EndpointAddress.ToLower());
```

Comment:
Just one nit -- can you also verify for live metrics?

Reply:
Talked with @gzuber -- this isn't completely straightforward so we'll validate with e2e testing.